torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1339,6 +1341,71 @@ def call_function(
         )
 
 
+class StrictModeHigherOrderVariable(TorchHigherOrderOperatorVariable):
+    def call_function(
```

Comment:
Can annotate the call_function with raise_hard_error_if_graph_break, so that any mutations/graph breaks will be turned into hard error like what we did for cond.

Reply:
Done